caddyconfig/httpcaddyfile/directives.go

```diff
@@ -70,6 +70,7 @@ var directiveOrder = []string{
 	"handle",
 	"handle_path",
 	"route",
+	"handle_errors",
```

Comment:
Ah, remove this line please.

Reply:
Just did. I forgot to pull changes from the remote branch after your commit. Thanks!